h/api/models/document.py

```diff
@@ -38,16 +38,19 @@ class Document(Base, mixins.Timestamps):
                                backref='document',
                                order_by='DocumentMeta.updated.desc()')
 
+    meta_title = sa.orm.relationship('DocumentMeta',
+                                     primaryjoin='and_(Document.id==DocumentMeta.document_id, DocumentMeta.type==u"title")',
+                                     order_by='DocumentMeta.updated.asc()',
+                                     uselist=False,
+                                     viewonly=True)
+
```

Comment:
Given the join condition, you're always going to have the possibility of there being multiple rows in this queryset. I'd suggest just accepting this and changing it to `meta_titles` with `uselist=True`.